docs/sources/tempo/getting-started/tempo-in-grafana.md

```diff
@@ -62,3 +62,14 @@ The JSON data can be downloaded via the Tempo API or the [Inspector panel](/docs
 {{% admonition type="note" %}}
 To perform this action on Grafana 10.1 or later, select a Tempo data source, select **Explore** from the main menu, and then select **Import trace**.
 {{% /admonition %}}
+
+## Link tracing data with profiles
+
+Using Trace to profiles, you can use Grafana’s ability to correlate different signals by adding the functionality to link between traces and profiles.
+
+Trace to profiles lets you link your [Grafana Pyroscope](/docs/pyroscope/) data source to tracing data in Grafana or Grafana Cloud.
+When configured, this connection lets you run queries from a trace span into the profile data.
+
+For more information, refer to the [Traces to profiles documentation](docs/grafana/latest/datasources/tempo/configure-tempo-data-source#trace-to-profiles) and the [Grafana Pyroscope data source documentation](/docs/grafana/latest/datasources/grafana-pyroscope/).
```

Comment:
@jdbaldry  I am not sure why doc-validator is saying I have to use a relref shortcode for this URL. It started as a full URL, which is what writers' toolkit says to do. doc-validator said nope, change to /docs/..., so I did. Now it says, use a relref. 

What's the right thing to use here?

Reply:
https://github.com/grafana/tempo/pull/3331

`doc-validator` 4.0.0 has the latest guidance. The current version in this repository's CI was released before the guidance changed.

https://grafana.com/docs/writers-toolkit/write/links/ is the correct guidance.